### PR TITLE
feat(sourcmaps): Add create-react-app option

### PR DESCRIPTION
**Pull request by lforst**
Adds a basic flow for CRA based on the sentry-cli flow.

### CODE REVIEW

File: src/sourcemaps/sourcemaps-wizard.ts [hunk 1]
```diff
@@ -190,8 +197,11 @@ async function startToolSetupFlow(
     case 'tsc':
       await configureSentryCLI(options, configureTscSourcemapGenerationFlow);
       break;
-    default:
+    case 'create-react-app':
       await configureSentryCLI(options);
       break;
+    default:
+      await configureSentryCLI(options, configureCRASourcemapGenerationFlow);
```
Comment (matejminar) [inline src/sourcemaps/sourcemaps-wizard.ts:204]: I might be missing some context - shouldn't this second param go into the `create-react-app` case?

Comment (lforst) [reply src/sourcemaps/sourcemaps-wizard.ts:204]: Oh you're totally right!

**Review by Lms24 — APPROVED** [review]
Nice!

**Review by matejminar — APPROVED** [review]
Looks good to me. I also like the change in wording regarding the access to your config 👏

**Pull request merged**